tests/_serialization/test_container_properties.py

```diff
@@ -53,9 +54,24 @@ def test_mapping_property_list_of_pairs(key_type, value_type, key_value, value_v
     assert prop.deserialize(serialized) == value
     unittest.TestCase().assertCountEqual(prop.serialize(value), serialized)
 
+
 def test_mapping_property_list_of_pairs_multiple():
     prop = properties.Mapping(properties.String, properties.Integer, ser_as_list_of_pairs = True)
     value = {'foo': 1, 'bar': 2}
     serialized = [('foo', 1), ('bar', 2)]
     assert prop.deserialize(serialized) == value
     unittest.TestCase().assertCountEqual(prop.serialize(value), serialized)
+
+class dummyDescriptor(object):
```

Comment:
```suggestion

class DummyDescriptor(object):
```
nit: pep spacing and camel case.